tracer/src/Datadog.Trace/Logging/Internal/DatadogLoggingFactory.cs

```diff
@@ -30,57 +29,65 @@ internal static class DatadogLoggingFactory
     private const int DefaultRateLimit = 0;
     private const int DefaultMaxLogFileSize = 10 * 1024 * 1024;
 
+    internal const string DefaultConsoleMessageTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] {Message:lj} {Exception}{NewLine}";
```

Comment:
Curious, how obvious is it that these log messages are coming from the tracer?
Wondering about the scenario that our logs end up in the same console stream as application logs and maybe it would be difficult to discern that they were from the Tracer?

So maybe adding "Datadog" or something in the template?

Reply:
I added `dd-trace-dotnet`. WDYT?

Reply:
Funny, this is all literally part of the reason we never shipped this 😅 

Also - FWIW, the serverless lambda integration _already_ writes to the console, and it uses this format, so we should probably unify these?

https://github.com/DataDog/dd-trace-dotnet/blob/7ed850c6adf7c4a6603b7f27630f5460fee500d4/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs#L127-L133

Reply:
> we should come back and change this to use the built-in logger

I am adding a TODO in the #7160 PR description to follow up on this.